ci-operator/step-registry/hypershift/kubevirt/run-e2e-external/hypershift-kubevirt-run-e2e-external-chain.yaml

```diff
@@ -78,7 +78,9 @@ chain:
                   --e2e.kubevirt-infra-kubeconfig=${RESTRICTED_INFRA_KUBECONFIG} \
                   ${STORAGE_CLASS_FLAG} \
                   --e2e.kubevirt-infra-namespace=${EXTERNAL_INFRA_NS} \
-                  --e2e.annotations=hypershift.openshift.io/management-platform=${INFRA_PLATFORM_TYPE}
+                  --e2e.annotations=hypershift.openshift.io/management-platform=${INFRA_PLATFORM_TYPE} \
+                  --e2e.service-cidr="172.33.0.0/16" \
+                  --e2e.cluster-cidr="10.140.0.0/14"
```

Comment:
this is going to fail for backports because there are no cidr range options in previous branches for the test driver

would it be reasonable to change the AWS's cluster cidr range instead so it would work consistently for us across all branches?

another option would be to make these env vars to the chain that would only get set for main branch today.

Reply:
These are the CIDRs being used in this workflow:
1. ROSA HCP (aws infra) - service: 172.30.0.0/16, cluster: 10.128.0.0/14
2. AWS MGMT - service: 172.29.0.0/16, cluster: 10.132.0.0/14
3. KubeVirt MGMT (hosted on # 2): service: 172.32.0.0/16, cluster: 10.136.0.0/14
4. test hcp kv clusters (CP on # 3, VMs on # 2): service: 172.33.0.0/16, cluster: 10.140.0.0/14

I can try to change cluster # 2 to use a cluster CIDR of 10.124.0.0/14 instead of the hypershift default (132) and then the test clusters will be able to work with 172.31 and 10.132 (defaults).